sw/device/tests/crypto/cryptotest/firmware/trigger_sca.c

```diff
@@ -27,7 +27,7 @@ status_t handle_trigger_sca_select_source(ujson_t *uj) {
   cryptotest_trigger_sca_source_t uj_trigger;
   TRY(ujson_deserialize_cryptotest_trigger_sca_source_t(uj, &uj_trigger));
 
-  sca_select_trigger_type((sca_trigger_type_t)uj_trigger.source);
+  sca_select_trigger_type((*uj_trigger.source) & 0xFF);
```

Comment:
Why is `source` an array in the struct definition? also it's an array of 8-bit bytes so the AND is unecessary. I suggest writing `ut_trigger.source[0]`, or even better to change the definition to a scalar field unless there is a good reason why it should be one-element array.